### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by nwbirnie**

### CODE REVIEW

**Review by jradcliff — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+16.0.1 - 2021-11-24
```
Comment (jradcliff) [inline CHANGELOG.md:1]: Per chat, let's move this to next week and update the date here.

**Review by jradcliff — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nwbirnie) [reply CHANGELOG.md:1]: Code freeze is now over, please can you review/approve?

Comment (jradcliff) [reply CHANGELOG.md:1]: Could you update the date on line 1?

Thanks

Comment (nwbirnie) [reply CHANGELOG.md]: Done!

**Review by jradcliff — APPROVED** [review]

**Pull request merged**